### PR TITLE
Add truncate helper

### PR DESCRIPTION
**Pull request by romulomachado**
### Example

```handlebars
{{truncate "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas hendrerit quam enim, in suscipit est rutrum id. Etiam vitae blandit purus, sed semper sem." 140}}
```

returns

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas hendrerit quam enim, in suscipit est rutrum id. Etiam vitae blandit purus,...
```

### CODE REVIEW

**Comment by skeate** [comment]
Should this truncate to x-3? To account for the ellipsis.

**Comment by romulomachado** [comment]
@skeate Updated! https://github.com/DockYard/ember-composable-helpers/pull/135/commits/d58f603e23ba75095dbd35eb8705616e44de8266

**Comment by poteto** [comment]
This is failing due to Travis requiring the use of it's Trusty builds for installing Google Chrome. This will be fixed in #137, I will let you know when you can rebase this PR

**Comment by poteto** [comment]
Could you also please add an entry into [`index.js`](https://github.com/DockYard/ember-composable-helpers/blob/master/addon%2Findex.js)? Thanks!

File: addon/helpers/truncate.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { helper } from 'ember-helper';
+
+export function truncate([string, characterLimit = 140]) {
+  let limit = characterLimit - 3;
+
+  if (string) {
+    if (string.length > limit) {
```
Comment (poteto) [inline addon/helpers/truncate.js]: Could just be `if (string && string.length > limit)` to remove the double `if`s

Comment (romulomachado) [inline addon/helpers/truncate.js]: Done! https://github.com/DockYard/ember-composable-helpers/pull/135/commits/60dfb4613848facb84ee55029c58d0930e2c723d

**Comment by romulomachado** [comment]
Just added! https://github.com/DockYard/ember-composable-helpers/pull/135/commits/3891c8dc295e413a083f9eab36060c3502981210

**Comment by poteto** [comment]
You can rebase against master now :)

**Comment by romulomachado** [comment]
@poteto Done! 😄

**Comment by poteto** [comment]
@homu r+

**Pull request merged**